core/src/main/java/org/apache/calcite/plan/volcano/RelSubset.java

```diff
@@ -433,6 +433,13 @@ RelNode buildCheapestPlan(VolcanoPlanner planner) {
     return list;
   }
 
+  /**
+   * Checks whether the rel is a part of the current subset.
+   */
+  public boolean containsRel(RelNode node) {
```

Comment:
change the name to `contains` rather than `containsRel`?

Change the javadoc to 'Returns whether' rather than 'Checks whether'. (Often methods that 'check' return void, and throw if the condition is violated.)

Reply:
done